pkgs/top-level/all-packages.nix

```diff
@@ -10227,6 +10227,9 @@ in
 
   nsis = callPackage ../development/tools/nsis { };
 
+  ### DEVELOPMENT / TOOLS / VERSION CONTROL
+  lefthook = callPackage ../development/tools/version-control/lefthook {};
```

Comment:
I would prefer this to be placed under `### DEVELOPMENT / EMSCRIPTEN`; because `pre-commit` (same category) is under there